### PR TITLE
Ensures there are Potatoes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
- Fixes there not being a pre-defined potato grown
- Fixes there being an attackby for potatoes that's handled elsewhere...properly 

Fixes https://github.com/ParadiseSS13/Paradise/issues/3737

### CODE REVIEW

**Comment by Tastyfish** [comment]
:+1: @Crazylemon64 literally had to farm potatoes the old fashioned way in the admin testing area to have adminbus potatoes.

**Comment by Dumbdumn5** [comment]
:+1: Prince Ferdinand McSpudington IV is back!

**Comment by taukausanake** [comment]
So we can't cut potatoes? I guess it doesn't matter but okay

**Comment by Fox-McCloud** [comment]
@taukausanake 

No, you can cut potatoes---this is old code for an older styled system we don't use. *actual* potato cutting is handled elsewhere.

File: code/modules/hydroponics/grown_predefined.dm [hunk 1]
```diff
@@ -6,6 +6,9 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosiavulgaris
 	plantname = "ambrosia"
 
+/obj/item/weapon/reagent_containers/food/snacks/grown/potato
+	plantname = "potato"
+
```
Comment (Crazylemon64) [inline code/modules/hydroponics/grown_predefined.dm]: Other possibly useful fruits that lack an entry here:
* Nettles
* Golden apples
* Wheat
* Bluespace tomatoes
* Money pods
* Nova flowers

**Pull request merged**